eclass/elisp-common.eclass

```diff
@@ -165,7 +166,7 @@
 # to above calls of elisp-site-regen().
 
 case ${EAPI:-0} in
-	4|5|6) inherit eapi7-ver ;;
+	5|6) inherit eapi7-ver ;;
 	7) ;;
 	*) die "${ECLASS}: EAPI ${EAPI:-0} not supported" ;;
 esac
```

Comment:
no inherit guard?

Reply:
These are "leaf" eclasses that aren't (passively) inherited by any other eclass. And in EAPI 7 they also don't (actively) inherit anything else. So no danger of cycles, and I think we can save these variables from the environment.